src/Avalonia.Visuals/Platform/IPlatformRenderInterface.cs

```diff
@@ -168,6 +168,9 @@ public interface IPlatformRenderInterface
         /// <returns></returns>
         IGlyphRunImpl CreateGlyphRun(GlyphRun glyphRun);
 
+        /// <summary>
+        /// Gets a value indicating whether the platform supports rectangles with rounded corners.
+        /// </summary>
         bool SupportsIndividualRoundRects { get; }
```

Comment:
Someone needs to check my comment here as it may not be entirely correct. The purpose of this property is unclear and it is largely unused in code. I'm not sure if "IndividualRoundRects" means each corner can be individually rounded or it means each rectangle can have rounded corners. (assuming the first).

Reply:
>  each corner can be individually rounded 

This is closer to what @kekekeks meant i think.

Reply:
Either Cairo or Direct2D couldn't render rounded rects. So our controls that require rounded corners generate a geometry instead. Said geometry has to be retained, so it lives in the control itself.

Reply:
ok, will slightly adjust the comment.

I think Direct2D is still maintained to some degree but Cairo is removed now, correct? if this was for Cairo perhaps the property itself should be removed soon.

Reply:
Updated the comment.